package.json

```diff
@@ -2,7 +2,8 @@
   "name": "bayesjs",
   "version": "0.4.3",
   "description": "Inference on Bayesian Networks",
-  "main": "dist/bayes.js",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
```

Comment:
You can keep `main` as `dist/index.js` since webpack creates this result:

<img width="580" alt="Screen Shot 2020-03-23 at 10 12 49" src="https://user-images.githubusercontent.com/2437673/77320842-0cbec080-6cf0-11ea-9fda-83d662277b2d.png">

Reply:
True, changed this.